src/proof.rs

```diff
@@ -529,9 +529,9 @@ impl<'a, 'b> ProofTreePrinterImpl<'a, 'b> {
                         if frame.var_list[..frame.mandatory_count].contains(&atom) {
                             continue;
                         }
-                        &mut proof_ordered_hyps
```

Comment:
Floating hypothesis which are not mandatory are normally added to the `proof_ordered` list, to be listed in the parens. This occurs for variables which are present neither in essential hypotheses nor in the final statement, but nevertheless used in the proof.